package.json

```diff
@@ -20,7 +20,11 @@
   "private": true,
   "dependencies": {
     "@axe-core/playwright": "^4.11.0",
+    "@code-pushup/cli": "^0.96.1",
+    "@code-pushup/models": "^0.96.1",
+    "@code-pushup/nx-plugin": "^0.96.1",
```

Comment:
We have to install it because ATM Nx plugins don't take import aliases in any way. This means our nx-plugin code is always 1 release behind the current PR